### DISCUSSION TITLE
How do I use the projector without a linux session starting on it

### DISCUSSION THREAD

**Discussion by Ryan-Te** [fnuecke/oc2#171] [Q&A]
I want to control it from the primary session, but the cursor from the projectors session is still on there
![image](https://user-images.githubusercontent.com/46229874/162596437-e06c151a-6764-41d8-a091-58ea80d4fcbe.png)

**Comment by tobi-blackscale** [comment]
No idea how to stop the session on it, but you can get rid of the cursor with:

`echo 0 > /sys/class/graphics/fbcon/cursor_blink`
or
`echo -e '\033[?17;0;0c' > /dev/tty1`
 (might be a different tty depending how many projectors are connected)
 
 That should do for most cases.

**Comment by untodesu** [comment]
A projector is a bus device, so you will need some kind of custom kernel to operate it. You either wait for custom kernel support (#127) or for community to make a glue mod or something else to link CC:tweaked with OC2 (I'm sure there are crazy people ready to write that kind of mod)

**Reply by tobi-blackscale** [reply]
The framebuffer works, you can even display images with it, just by using the device file. Not comfortable, but definitely doable without a custom kernel or some glue mod.  
![grafik](https://user-images.githubusercontent.com/71216944/182146578-24d1c69b-be66-4f56-bccc-7f35b14107eb.png)

**Reply by untodesu** [reply]
Ah I thought you were interested in using a projector without _Linux at all_ but you were referring to a terminal session on it, right
